wolfcrypt/src/random.c

```diff
@@ -3655,15 +3652,6 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
         return 0;
     }
 
-#elif defined(WOLFSSL_RENESAS_TSIP)
```

Comment:
I don't want this removed. I added it back for a customer in ZD 18222. The TSIP RNG comes out of a CTR-DRBG but there is no issue using it as a seed for our DRBG. If a customer wants to use it directly then they can still define `CUSTOM_RAND_GENERATE_BLOCK` to override our DRBG and use `wc_tsip_GenerateRandBlock` directly.